### PR TITLE
Enterprise Console & accessToken docs

### PR DESCRIPTION
**Pull request by calebhailey**

### CODE REVIEW

**Comment by agoddard** [comment]
@calebhailey \o/ one comment - dashboard/console terminology feels a bit mixed - are we settling on one, or are we defining them elsewhere?

**Comment by calebhailey** [comment]
@agoddard over the weekend I settled on "Dashboard" = the dashboard as we know it (the web app), and "Console" = the dashboard backend API (i.e. the federated API w/ the new token-based access controls, etc). This will need to be updated in other places in the website as well (e.g. /enterprise page, etc). 

Thoughts?

File: docs/0.26/enterprise/api.md [hunk 1]
```diff
@@ -32,6 +33,22 @@ of several endpoints to provide access to data for generating reports,
 visualizing internal metrics, and more. The Enterprise API also supports native
 SSL, for end-to-end SSL, eliminating the need for a proxy to terminate SSL.
 
+## What is the Sensu Enterprise Console?
+
+The Sensu Enterprise Console is a federated API endpoint provided by the [Sensu
+Enterprise Dashboard][4] for API access to multiple [Sensu datacenters][5]
+(available in Sensu Enterprise Dashboard version 1.10 and newer).
```
Comment (calebhailey) [inline docs/0.26/enterprise/api.md]: This probably needs some more words to explain how to access this API... e.g. is it available at the same port as Uchiwa (3000 by default), or is there some other port?

File: docs/0.26/enterprise/rbac/overview.md [hunk 2]
```diff
@@ -48,6 +50,17 @@ RBAC:
 * [GitHub](rbac-for-github.html)
 * [GitLab](rbac-for-gitlab.html)
 
+## RBAC for the Sensu Enterprise Console API
+
+As of Sensu Enterprise Dashboard version 1.12, RBAC configurations may be
+applied to the [Sensu Enterprise Console API][?]. Access to the Sensu Enterprise
```
Comment (portertech) [inline docs/0.26/enterprise/rbac/overview.md]: ? link

**Pull request merged**